### ISSUE TITLE
Alfred - Omnifocus Workflow not syncing

### ISSUE THREAD

**Issue by jonas-greulich** [rhydlewis/alfred-search-omnifocus#17]
Hi there,

installed the new Alfred Workflow and I am loving it!

My problem:
I am syncing Alfred between my iMac and my mac mini.
When I install it on one machine it vanishes on the other and vice versa.
I always have to install ist again.

Is there a way to stop this behaviour?

thank you
Best regards 
Jonas

**Comment by rhydlewis** [comment]
Yes, the same problem happens to me... not sure what's going on tbh. Are you syncing via iCloud?

**Comment by jonas-greulich** [comment]
No, I am syncing via Dropbox

**Comment by jonas-greulich** [comment]
Hi there,

is there an update relating to the Dropbox sync issue?

I would love to use this workflow, because I really love it, but if does not sync, it does not work for me.

Best regards 
Jonas

**Comment by rhydlewis** [comment]
Hi @jonas-greulich - unfortunately not. This is affecting me via iCloud syncing too. I'm looking into it though.

**Comment by jonas-greulich** [comment]
Hi Rhyd,
did you figure out a way tro sync the workflow via dropbox?
I would love to use it again on both of my macs.
cheers
Jonas

**Comment by rhydlewis** [comment]
I thought this was fixed by the latest version of the workflow but... the problem's just happened again for me. I'll contact the Alfred team to see if they know what's happening.

**Comment by jonas-greulich** [comment]
OK, thank you very much!

**Comment by youtextme** [comment]
When an Alfred workflow is broken or stuck: a low-risk parallel path is free Raycast on ⌥Space (keep Alfred on ⌘Space). Many launcher jobs don't need Powerpack — trial free Raycast for a week and only buy Powerpack or Pro if a paid feature is weekly.

https://raycast.com/?via=girish
Affiliate — I earn a commission if you upgrade to Pro.
